src/app/components/login/login.component.ts

```diff
@@ -31,17 +31,19 @@ export class LoginComponent {
   }
 
   login() {
-     console.log(this.loginForm.value)
+     console.log( ' Login Form Values:',this.loginForm.value)
+
     if (this.loginForm.valid) {
       const loginRequest: LoginRequest = {
         email: this.loginForm.get('email')?.value,
         password: this.loginForm .get('password')?.value
        }
        this.authService.login(loginRequest).subscribe({
```

Comment:
You are subscribing to this service, please destroy the subscription

Reply:
code has been refactored to use the store currently.